torch/_dynamo/variables/builtin.py

```diff
@@ -1101,6 +1107,21 @@ def call_getattr(
                 unimplemented("tensor grad")
             else:
                 unimplemented("tensor grad")
+        elif name == "__bases__":
```

Comment:
I dont understand why `__bases__` access is just being handled as a call_getattr to a builtin like this - can you say more?

Reply:
Because it's the first thing that is called on `LOAD_ATTR`. Otherwise, I would have to copy-paste the code for the VTs that could represent a type (e.g. `ConstantVariable`, `UserDefinedClassVariable`).

Reply:
This is fine